src/zope/structuredtext/docbook.py

```diff
@@ -282,7 +282,9 @@ def read(self):
                '<book>\n')
         out = out + '<title>%s</title>\n' % self.title
         for chapter in self.chapters:
-            out = out + chapter + '\n</book>\n'
+            out = out + chapter + '\n'
+
+        out += '\n</book>\n'
```

Comment:
Should this be mentioned in the changelog as bugfix?

Reply:
Yup, thanks.